tt_metal/impl/dispatch/command_queue_interface.hpp

```diff
@@ -210,6 +218,9 @@ struct dispatch_constants {
     uint32_t prefetch_d_buffer_pages_;
     uint32_t dispatch_s_buffer_size_;
     std::vector<uint32_t> device_cq_addrs_;
+    static inline std::unique_ptr<dispatch_constants> inst = nullptr;
+    static inline uint32_t hw_cqs = 0;
```

Comment:
Can you remove the init values here please.

Reply:
Done